replica/table.cc

```diff
@@ -1223,7 +1224,8 @@ void table::set_metrics() {
                 ms::make_gauge("pending_compaction", ms::description("Estimated number of compactions pending for this column family"), _stats.pending_compactions)(cf)(ks),
                 ms::make_gauge("pending_sstable_deletions",
                         ms::description("Number of tasks waiting to delete sstables from a table"),
-                        [this] { return _sstable_deletion_sem.waiters(); })(cf)(ks)
+                        [this] { return _sstable_deletion_sem.waiters(); })(cf)(ks),
+                ms::make_gauge("tablet_count", ms::description("Tablet count"), _stats.tablet_count)(cf)(ks)
```

Comment:
It should be also under enable_node_aggregated_table_metrics, otherwise it's not exported by default.

Reply:
> It should be also under enable_node_aggregated_table_metrics, otherwise it's not exported by default.

Updated the patch to contain the mentioned changes :+1: